### PR TITLE
dbeaver/pro#2712 Added a shortcut for advance copy with last settings

### PR DESCRIPTION
**Pull request by HocKu7**
Added a shortcut Ctrl+Shift+Alt+C for an advance copy with the last settings. If the last settings are undefined (e.g. at the first time) then it should work as a usual advance copy. Added hint on the form.
image

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties [hunk 1]
```diff
@@ -450,6 +450,7 @@ copy_special_force_quote_cell_values_text = Always quote values
 copy_special_force_quote_cell_values_tip = Place all cell values in quotes
 copy_special_copy_as_html_text = Copy as HTML
 copy_special_copy_as_html_tip = Copy as HTML (in addition to plaintext format)
+copy_special_hint_for_hotkey = For future use with the same settings, you\nmay use the hotkey: {0}
```
Comment (serjiokov) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties:453]: I would recommend to use
`The advanced copy settings are saved now, please use the key: {0} to invoke it`

**Review by serjiokov — APPROVED** [review]

**Comment by MashaKorax** [comment]
https://github.com/user-attachments/assets/67519303-1ced-47b3-b881-de79a7ae583c

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/handler/ResultSetHandlerCopySpecial.java [hunk 2]
```diff
@@ -55,7 +55,8 @@ public class ResultSetHandlerCopySpecial extends ResultSetHandlerMain implements
     public static final Log log = Log.getLog(ResultSetHandlerCopySpecial.class);
     public static final String CMD_COPY_SPECIAL = IActionConstants.CMD_COPY_SPECIAL;
     public static final String CMD_COPY_SPECIAL_LAST = IActionConstants.CMD_COPY_SPECIAL_LAST;
-    private AdvancedCopyConfigDialog lastConfigDialog = null;
+    //private AdvancedCopyConfigDialog lastConfigDialog = null;
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/handler/ResultSetHandlerCopySpecial.java]:
```suggestion
```

**Comment by MashaKorax** [comment]
verified

**Pull request merged**